test/cases/Global.webidl

```diff
@@ -1,4 +1,4 @@
-[Global]
+[Global=Global]
```

Comment:
Should we throw an error if the RHS is absent?

Reply:
We don't really claim to support `[Global]` in the README, but sure.

Reply:
Hmm yeah, I'm a little confused what the state of our support is there. (For [Global], not necessarily [Global] + named properties.)